#! /usr/bin/env python

PACKAGE='ds_motion_generator'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name     Type    Level Description     	Default 	Min   	Max
gen.add("Wn",                double_t,    0,    				
	"Speed of the", 							 10,     	0, 		100		)
gen.add("fil_dx_lim" ,  double_t,    0,    				
	"Velcoity limit", 	   						 1,    		0, 		5		)
gen.add("fil_ddx_lim" ,   double_t,    0,    				    
	"Acc. limit", 	   						     5,     	0, 		10		)


gen.add("offset_x" ,   double_t,    0,    				    
	"X of attractor.", 	   						0,     	-.5, 		.5		)
gen.add("offset_y" ,   double_t,    0,    				    
	"Y of attractor.", 	   						0,     	-.5, 		.5		)
gen.add("offset_z" ,   double_t,    0,    				    
	"Z of attractor.", 	   						0,     	-.5, 		.5		)

gen.add("scaling" ,   double_t,    0,    				    
	"scaling the velocities",    	          	1,     	  0, 		 5		)

gen.add("trimming" ,   double_t,    0,    				    
	"trimming the velocities",					.2,      0, 		 .5		)


exit(gen.generate(PACKAGE, "ds_motion_generator", "SED_params"))